src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -0,0 +1,2394 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+
+namespace Microsoft.CodeAnalysis.Semantics
```

Comment:
that is last step on my work.
"after that, I will create xml/generator from Operation.xml.Generated.cs which can produce Operation.xml.Generated.cs"

I dont have generator yet. I wrote custom tool to create this file from IOperation interfaces. I still used the file name I will use at the end.